x/auth/types/account.go

```diff
@@ -408,9 +412,10 @@ type GenesisAccount interface {
 // custom json marshaler for BaseAccount & ModuleAccount
 
 type BaseAccountJSON struct {
-	Address  string          `json:"address"`
-	PubKey   json.RawMessage `json:"pub_key"`
-	Sequence string          `json:"sequence"`
+	Address  string `json:"address"`
+	PubType  byte   `json:"pub_type"`
+	PubKey   []byte `json:"pub_key"`
```

Comment:
How about grouping the pubkey properties? 

e.g.
```
{
  "address": ...,
  "sequence": ...,
  "pub_key": {
    "type":...,
    "key":...,
  }
}
```

Reply:
Okay, I apply it.